server/chat-commands.js

```diff
@@ -2598,7 +2598,7 @@ const commands = {
 		if (!targetUser && !room.log.hasUsername(target)) return this.errorReply(`User ${target} not found or has no roomlogs.`);
 		if (!targetUser && !user.can('lock')) return this.errorReply(`User ${name} not found.`);
 		let userid = toId(this.inputUsername);
-		if (!this.can('mute', targetUser, room)) return;
+		if (!(user === targetUser && user.can('mute', null, room)) && !this.can('mute', targetUser, room)) return;
```

Comment:
```suggestion
```

Reply:
Isn't this just completely removing the permission check?
do you mean ``if (!this.can('mute', null, room)) return;``?

Reply:
Oh yes, I did mean that.

Reply:
```suggestion
		if (!this.can('mute', null, room)) return;
```